### ISSUE TITLE
Consider using `pyproject.toml`

### ISSUE THREAD

**Issue by audiodude** [noisebridge/MediaBridge#42]
Probably can't use a `pyproject.toml` for dependencies, since we are using `pipenv`.

It might be useful for specifying config and metadata for other parts of the project.

**Comment by jhanley634** [comment]
Yes, this is helpful for linters. It has been in effect since January 21st, 6060dec8.
